src/Features/Core/Portable/SignatureHelp/AbstractSignatureHelpProvider.cs

```diff
@@ -277,5 +287,21 @@ private SignatureHelpItem UpdateItem(SignatureHelpItem item, SupportedPlatformDa
 
             return supportedPlatforms;
         }
+
+        protected static int? GetSelectedIndex<TSymbol>(ImmutableArray<TSymbol> candidates, SymbolInfo currentSymbol) where TSymbol : class, ISymbol
```

Comment:
:bulb: `TryGetSelectedIndex` (since it returns `null` if the value was not found)